games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.6.6.0.47291.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop prefix unpacker wrapper
+
+DESCRIPTION="Baldur's Gate: Enhanced Edition"
+HOMEPAGE="https://www.baldursgate.com/"
+SRC_URI="baldur_s_gate_enhanced_edition_${PV//./_}.sh"
+
+LICENSE="GOG-EULA"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+RESTRICT="bindist fetch"
+
+RDEPEND="dev-libs/expat
+	dev-libs/openssl-compat
+	media-libs/openal
+	virtual/opengl
+	x11-libs/libX11"
+BDEPEND="app-arch/unzip"
+
+QA_PRESTRIPPED="/opt/${PN}/BaldursGate"
```

Comment:
`QA_PRESTRIPPED` should not start with a `/`:

[QA_PRESTRIPPED](https://devmanual.gentoo.org/eclass-reference/ebuild/index.html)
 
> This should contain a list of file paths, relative to the image directory, of files that contain pre-stripped binaries. The paths may contain regular expressions with escape-quoted special characters.

Reply:
I agree with all the others, but I am not sure about this change. This line is already in the 2.5.23121 ebuild (which I copied to make my update) and points to the final install location in `/opt`.

If the slash is removed then it will refer to a relative `opt` folder which does not exist in the game folder.

Reply:
It should be relative to the image directory (default `/var/tmp/portage/${CATEGORY}/${PVR}/image`).  I think it works like this too because the QA script removes the starting / if needed, but most ebuilds don't have it.

Reply:
No worries then. I just recompiled without the `/` and it still works.

Reply:
Excellent.  Thank you!